src/clj/battlebots/arena/generation.clj

```diff
@@ -37,31 +38,86 @@
   [amount item arena]
   (reduce replacer arena (repeat amount item)))
 
+(defn- wrap
```

Comment:
@cch1 `wrap` & `move` are very well written. I have two functions in `arena.utils`, `wrap-coords` & `adjust-coords` that accomplish a similar goal, but I'd like to replace them with yours. The signatures are similar, the biggest difference I can see is that `adjust-coords` allows you to apply a distance to adjust, however always wraps. `move` appears to always adjust one space, but allows for an oob coord to be calculated (defaults to wrap)? I wired the tests I wrote for `wrap-coords` up to `wrap` and they all pass without modification, so I believe we can just drop yours in it's place. What's your thoughts on `move` & `adjust-coords`?